_data/chains/eip155-2002.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Milkomeda A1 Mainnet",
+  "chain": "milkALGO",
+  "icon": "milkomeda",
+  "network": "mainnet",
```

Comment:
Just FYI that the `network` key was removed https://github.com/ethereum-lists/chains/pull/1508